records_mover/db/bigquery/unloader.py

```diff
@@ -33,7 +33,11 @@ def can_unload_format(self, target_records_format: BaseRecordsFormat) -> bool:
         return False
 
     def can_unload_to_scheme(self, scheme: str) -> bool:
-        return scheme == 'gs'
+        if scheme == 'gs':
+            return True
+        # Otherwise we'll need a temporary bucket configured for
+        # BigQuery to unload into
+        return self.gcs_temp_base_loc is not None
```

Comment:
With this we can unload via a GCS scratch bucket.  I forgot this in the last PR!